demisto_sdk/commands/content_graph/objects/integration.py

```diff
@@ -47,21 +48,17 @@ class Parameter(BaseModel):
     fromlicense: Optional[str] = None
 
 
-class Output(BaseModel):
-    description: str = ""
-    contentPath: Optional[str] = None
-    contextPath: Optional[str] = None
+class IntegrationOutput(Output):
     important: Optional[bool] = False
     importantDescription: Optional[str] = None
-    type: Optional[str] = None
 
 
 class Command(BaseNode, content_type=ContentType.COMMAND):  # type: ignore[call-arg]
     name: str
 
     # From HAS_COMMAND relationship
     args: List[Argument] = Field([], exclude=True)
-    outputs: List[Output] = Field([], exclude=True)
+    outputs: List[IntegrationOutput] = Field([], exclude=True)
```

Comment:
why do we separate outputs of integrations and scripts? isn't it the same?

Reply:
Not the same, integrations have 2 more fields (important, importantDescription)